src/utils.ts

```diff
@@ -147,6 +150,15 @@ export function getConfig(
     daffodilDebugClasspath: daffodilDebugClasspath
       ? daffodilDebugClasspath
       : defaultConf.get('daffodilDebugClasspath', ''),
+    dataEditorOmegaEditPort: dataEditorOmegaEditPort
+      ? dataEditorOmegaEditPort
+      : defaultConf.get('dataEditor.omegaEditPort', 9000),
```

Comment:
Can't we pull the default port from package.json?

Reply:
That is what this trying to do. The second value is the default value if nothing is found

Reply:
Shouldn't something _always_ be specified for a default server port (debug, data, or otherwise) in `package.json`.  If not, that's a bug in my opinion and the code should error out indicating that no default value was specified in `package.json` when it should have been.

Reply:
Sometimes you want a failsafe (e.g., PID files to help recover from an unexpected crash), and other times you want to _reveal_ programming errors (e.g., asserts).  In this case, I'd prefer the latter.

Reply:
I can see where you are coming from. But I believe maybe we should ask others for their input as well before deciding what to change to and do from now on. @stevedlawrence @mbeckerle @rt320 @arosien @stricklandrbls @michael-hoke @NolanMatt @nlewis05  what are your thoughts?

Reply:
I don't think I have a strong opinion one way or the other in general--there's good arguments for both sides. It's probably easier to detect typos or configuration name changes if you don't do defaults, so I'd maybe lean towards no default.

That said, in this particular case it looks like all uses of `defaultConf.get(...)` provide a default value, so at least in this PR I suggest we stay consistent with that and provide a default value.